API/Controllers/ProjectController.cs

```diff
@@ -239,6 +239,26 @@ public async Task<IActionResult> MigrateDatabaseToElasticSearch()
             return Ok(resultsResource);
         }
 
+        /// <summary>
+        ///     This method returns suggestions while searching for projects
+        /// </summary>
+        /// <param name="query"></param>
+        /// <returns>This method returns a list of autocomplete project resources.</returns>
+        /// <response code="200">This endpoint returns a list with suggested projects.</response>
+        [HttpGet("search/autocomplete")]
+        [ProducesResponseType(typeof(AutocompleteProjectsResource), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(ProblemDetails), (int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> GetAutoCompleteProjects([FromQuery(Name ="query")] string query)
+        {
+            List<Project> projects = await projectService.FindProjectsWhereTitleStartsWithQuery(query);
+            AutocompleteProjectsResource resultResource = new AutocompleteProjectsResource();
+            List<AutocompleteProjectResource> autocompleteProjectResources = mapper.Map<List<Project>, List<AutocompleteProjectResource>>(projects);
```

Comment:
We use most of the time [name]ResourceResult for return types and [name]Resource for the input data. Here you return [name]Resources, this could be confusing.

Reply:
Agree, refactored